src/lib/util/Config.ts

```diff
@@ -123,21 +123,29 @@ export class Config {
 			return true;
 		}
 
-		return !(await this.getConfigValue('disabled', engine, this.typeChecker.booleanCheck));
+		return !(await this.getBooleanConfigValue('disabled', engine));
 	}
 
-	public async getSupportedLanguages(engine: ENGINE): Promise<string[]> {
-		return await this.getConfigValue('supportedLanguages', engine, this.typeChecker.stringArrayCheck);
+	public getSupportedLanguages(engine: ENGINE): Promise<any> {
```

Comment:
These don't need to be async or await, they can just return the Promise from the delegate.

Reply:
Why did you switch these from `Promise<string[]>` to `Promise<any>`?

Reply:
Thanks, was a mistake, will fix.